bin/check_required_files_present

```diff
@@ -21,20 +22,45 @@ function require_file {
 	fi
 }
 
+function reject_file {
+	filename=$1
+	if [ -f "$filename" ]; then
+		echo "unneeded file should be removed: $filename" >&2
+		let "UNNEEDED_FILES+=1"
+	fi
+}
+
 function require_file_or_alternative {
 	filename=$1	
 	alternative_filename=$2
 	if [ ! -f $filename ] && [ ! -f $alternative_filename ]; then		
 		echo "required file missing: $filename or $alternate_filename" >&2
 		let "MISSING_FILES+=1"
```

Comment:
```suggestion
		MISSING_FILES=$(( MISSING_FILES +1 ))
```

Reply:
I am unable (as a matter of commit cleanliness) to include this change in this pull request, as it changes code that was not added in this pull request. This change will need to go in its own pull request.